packages/matter.js/src/behavior/internal/Reactors.ts

```diff
@@ -211,6 +211,10 @@ class ReactorBacking<T extends any[], R> {
         if (!(error instanceof Error)) {
             error = new Error(`${error}`);
         }
+        if (error instanceof MatterError) {
+            logger.error("Unhandled MatterError", this.#augmentError(error));
```

Comment:
If we rethrow I think we shouldn't also log or the stack trace will end up in logs twice.